fluid/object_detection/train.py

```diff
@@ -131,17 +131,18 @@ def test(pass_id):
     args = parser.parse_args()
     print_arguments(args)
     data_args = reader.Settings(
-        data_dir='./data',
+        dataset='coco', # coco or pascalvoc
+        data_dir='./data/coco',
         label_file='label_list',
         apply_distort=True,
         apply_expand=True,
         resize_h=300,
         resize_w=300,
         mean_value=[127.5, 127.5, 127.5])
     train(args,
-          train_file_list='./data/trainval.txt',
-          val_file_list='./data/test.txt',
-          data_args=data_args,
-          learning_rate=0.001,
-          batch_size=32,
-          num_passes=300)
+        train_file_list='./data/coco/annotations/instances_train2014.json',
+        val_file_list='./data/coco/annotations/instances_val2014.json',
```

Comment:
Do not specify the COCO data path here.  Should allow the user to specify data by the input arguments.